src/libcore/convert.rs

```diff
@@ -44,6 +77,21 @@ pub trait Into<T>: Sized {
 }
 
 /// Construct `Self` via a conversion.
```

Comment:
Seems like this line should be grammatically consistent with the first line of `Into`.